docs/en/docs/lakehouse/multi-catalog/jdbc.md

```diff
@@ -176,6 +164,7 @@ Currently, Jdbc Catalog only support to use 5.x version of JDBC jar package to c
 | `jdbc_url `     | Yes             |               | JDBC connection string                             |
 | `driver_url `   | Yes             |               | JDBC Driver Jar                                    |
 | `driver_class ` | Yes             |               | JDBC Driver Class                                  |
+| <version since="dev">`only_specified_database`</version> | No             |     "false"          | When the JDBC is connected, you can specify which database/schema to connect. For example, you can specify the DataBase in mysql `jdbc_url`; you can specify the CurrentSchema in PG `jdbc_url`, `only_specified_database` specifies whether only the database is specified simultaneously.                                  |
```

Comment:
do not support this type of `<version>`
you have to explain it below, as `driver_url`

Reply:
And you forgot the add `lower_case_table_names` doc

Reply:
done